Update Git Bash installation/setup instructions

libcce opened the pull request:
- New version of Git Bash
- Update to Git Bash installation/setup instructions
- Feedback from host site UNC Library Carpentry Oct 18
- Update thanks to @fmichonneau, also tagging for review

Commit:
Update Git Bash installation/setup instructions

- New version of Git Bash
- Update to Git Bash installation/setup instructions
- Feedback from host site UNC Library Carpentry Oct 18
- Update thanks to @fmichonneau, also tagging for review

fmichonneau commented on index.md:
```diff
@@ -308,6 +308,14 @@ and our administrator may contact you if we need any extra information.</h4>
 
+
+
+- Keep "Use Windows' default console window" selected and click on "Next".
```
I don't think you intended to have this paragraph here.

fmichonneau commented on index.md:
```diff
@@ -316,52 +324,46 @@ and our administrator may contact you if we need any extra information.</h4>
-          </ol>
-        </li>
-        <li>
-          If your "HOME" environment variable is not set (or you don't know what this is):
```
the setting up of the HOME environment variable should probably stay.

fmichonneau commented:
Thanks for putting this together @libcce ! see the  couple of inline comments.

Commit:
Update based on fmichonneau feedback

Update based on https://github.com/carpentries/workshop-template/pull/536/files/fc66f4018a8f29ace4c542dd888eb0a8a0c6948c

libcce commented:
Hi Francois, just coming around to this again since there has been a question about updating Git Bash setup on the discuss list. It was difficult for me to see what I should keep due to the formatting and also not having walked through the process myself. I believe these edits reflect the changes you commented on https://github.com/carpentries/workshop-template/pull/536/commits/8115f051bbe3c06f55d3c2d3d8bd660d535dae19.

fmichonneau commented:
these instructions are now outdated.
